File: Python/sequencia_de_numero_e_soma.py
m=int(input())
n=int(input())
total = 0 
while m > 0 and n > 0:

    if n < m:
        for w in range(n,m+1,1):
            print(w,end=" ")
            total += w
        print("Sum=",total)
    elif m < n:
        for w in range(m,n+1,1):
            print(w,end=" ")
            total += w
        print("Sum=",total)
        
    total=0
    m=int(input())
    n=int(input())

File: C/sequencia_de_numero_e_soma.cpp
#include<stdio.h>

int main(void){
	int m,n,total=0;
	
	scanf("%d %d", &m,&n);
	
	while(m > 0 && n > 0){
		if(n < m){
			for(int w = n; w <= m; w++){
				printf("%d ",w);
				total += w;
			}
			printf("Sum=%d\n",total);
		}
		else if(m < n){
			for(int w = m; w <= n; w++){
				printf("%d ",w);
				total += w;
			}
			printf("Sum=%d\n",total);
		}
		total = 0;
		scanf("%d %d", &m,&n);
	}
	
	return 0;
}
